app/models/europeana/record/annotations.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+##
+# Annotations support for records
+module Europeana
+  class Record
+    module Annotations
+      # TODO: handle pagination if more than 100 items
+      def annotations
+        @annotations ||= Annotation.find(annotations_search_params)
+      end
+
+      def annotations_search_params
+        {
+          qf: [
+            %(generator_name:#{annotations_api_generator_name}),
```

Comment:
Not sure if it makes sense in the context of this PR, or if it's limited by the Annotation API and how that stores the different kind of annotations, but it would be nice if this was more consistent and we used creator_name here too.

Reply:
`creator_name` is derived from the user token, `generator_name` from the API key.

For gallery annotations we only want to retrieve those created by the Collections gallery annotator, but on item pages we want annotations from potentially many users.

i.e. this needs to remain as it is.

Reply:
`creator_name` is derived from the user token, `generator_name` from the API key.

For gallery annotations we only want to retrieve those created by the Collections gallery annotator, but on item pages we want annotations from potentially many users.

i.e. this needs to remain as it is.